tensorflow/lite/kernels/activations.cc

```diff
@@ -347,6 +347,26 @@ TfLiteStatus ReluEval(TfLiteContext* context, TfLiteNode* node) {
   }
 }
 
+namespace {
+template <typename T>
+void QuantizedRelu1(const TfLiteTensor* input, TfLiteTensor* output) {
+  ActivationParams params;
+  int32 kMin = -1;
+  int32 kMax = 1;
+  params.activation_type = FusedActivationFunctionType::kRelu1;
+  // Relu1 has a min raneg of -1, we need to quantize this
+  params.quantized_activation_min =
+      (kMin) / input->params.scale + input->params.zero_point;
```

Comment:
These quantized_activation_min and quantized_activation_max values are used to clamp the output integer value. To be safe we should also ensure that these values are within the int32 range (the type of quantized_activation_min).
So maybe add something like what is done for QuantizedRelu6 in this file (clamping to the numerical limits )

Reply:
@suharshs , thanks for the review, i have updated the code as per your comments, kindly check and approve.

Regards
Amit